docs/advanced/configuration/proxies-and-certificates.mdx

```diff
@@ -156,3 +157,58 @@ networks:
 :::tip
 Of particular note here is that both configurations explicitly define which network they are using, in this case "proxy", but it can be named anything. It just has to be the same across all apps for which Traefik is serving as a proxy. These are marked as external because the proxy network was manually created by running: `docker network create proxy` but this might be unnecessary depending on HOW exactly you are running Traefik. For example, if running [Traefik with Portainer](https://docs.portainer.io/advanced/reverse-proxy/traefik#deploying-in-a-docker-standalone-scenario), you can follow their official docs on how to set up Traefik and Portainer together, and you can just focus on the Homarr docker labels instead.
 :::
+
+## Securing Homarr with Caddy
+
+If you are using Caddy as the reverse proxy for your setup, your `docker-compose.yml` should look something like this:
+
+```yaml
+services:
+
+  homarr:
+    container_name: homarr
+    image: ghcr.io/ajnart/homarr:latest
+    restart: unless-stopped
+    volumes:
+      - ./homarr/configs:/app/data/configs
+      - ./homarr/icons:/app/public/icons
+      - ./homarr/data:/data
+    ports:
+      - 7575:7575
+    networks:
+      - proxy
```

Comment:
I think it's better to remove this and abstract it with something like // [ homarr installation ]
(with the network instruction) 

This way we don't introduce a new breaking point in case we want to change the default homarr docker-compose config

Reply:
You mean that I have to use this placeholder to make a reference to any Docker-Compose configuration as this may change in the future. But I should mention that it must be on the same network as the Caddy service. Have I understood it correctly? If that's correct, where do I mention the network thing?

![20240709_20h52m08s_grim](https://github.com/homarr-labs/documentation/assets/61917364/9b57ff35-8cc3-4747-9842-759713776eed)

Do you like this way?

Reply:
> I think it's better to remove this and abstract it with something like // [ homarr installation ] (with the network instruction)
> 
> This way we don't introduce a new breaking point in case we want to change the default homarr docker-compose config

Or maybe this option?

![20240709_20h57m56s_grim](https://github.com/homarr-labs/documentation/assets/61917364/0f70620d-6985-4cc3-9720-b967ba9fec47)

Or this one 😄:
![20240709_20h58m59s_grim](https://github.com/homarr-labs/documentation/assets/61917364/f9805f12-c707-4c45-b452-7a7b11208703)

@ajnart pick the option you like the most and I will change this.

But one newbie question, how can I edit an existing Pull Request? I have never done it before...

Reply:
Last option is good but don't say "should" if it is required. 

"Homarr needs to be on the same network so you will need to add the following property to your homarr install: …"